doc/nrf/ug_thingy91.rst

```diff
@@ -0,0 +1,262 @@
+
++---------------+--------------------------------------------------+
+|Component      |  Build Target                                    |
++===============+==================================================+
+|nRF9160 SiP    |``nrf9160_pca20035`` for the secure version       |
+|               |                                                  |
+|               |``nrf9160_pca20035ns`` for the non-secure version |
++---------------+--------------------------------------------------+
+|nRF52840 SoC   |``nrf52840_pca20035``                             |
++---------------+--------------------------------------------------+
+
+You should use the build target ``nrf9160_pca20035`` or ``nrf9160_pca20035ns``, when building the application code for the nRF9160 SiP and the build target ``nrf52840_pca20035``, when building the application code for the onboard nRF52840 SoC.
+
+.. note::
+
+   LTE/GPS features can only be used with non-secure target.
+
+The table below shows the different types of build files that are generated and the different scenarios in which they are used:
+
++--------------+----------------------------------------+---------------------------------------+
+| File         | File format                            | Programming scenario                  |
++==============+========================================+=======================================+
+|merged.hex    | Full image, HEX format                 | Using debug probe                     |
++--------------+----------------------------------------+---------------------------------------+
+|app_signed.hex| MCUboot compatible image, HEX format   | Using nRF Connect Programmer          |
++--------------+----------------------------------------+---------------------------------------+
+|app_update.bin| MCUboot compatible image, binary format|* Using ``mcumgr`` command line tool   |
+|              |                                        |* For FOTA updates                     |
++--------------+----------------------------------------+---------------------------------------+
+
+There are multiple methods of programming a sample or application onto a Thingy:91.
+You can choose the method based on the availability or absence of an external debug probe to program.
+
+.. note::
+
+   In the event of not having an external debug probe to program the Thingy:91, you can directly program by :ref:`using the USB (MCUboot) method and nRF Connect Programmer<programming_usb>`.
+   The ``app_signed.hex`` firmware image file is used for the update in this scenario.
+
+.. _build_pgm_segger:
+
+Building and programming using SEGGER Embedded Studio
+=====================================================
+
+.. include:: gs_programming.rst
+   :start-after: build_SES_projimport_open_start
+   :end-before: build_SES_projimport_open_end
+
+
+.. figure:: images/ses_thingy_configuration.png
+   :alt: Opening the AT Client sample
+
+   Opening the AT Client sample for the nrf9160_pca20035ns build target
```

Comment:
Why are we using the AT Client sample here instead of the asset tracker?

Reply:
changed the screenshot and the example to 'Opening the Asset tracker application'